src/AuthService.Business/Commands/RefreshTokenCommand.cs

```diff
@@ -42,10 +42,11 @@ public LoginResult Execute(RefreshRequest request)
             var result = new LoginResult
             {
                 UserId = userId,
-                AccessToken = _tokenEngine.Create(userId, TokenType.Access),
-                RefreshToken = _tokenEngine.Create(userId, TokenType.Refresh)
+                AccessToken = _tokenEngine.Create(userId, TokenType.Access, out double tokenLifeTime),
+                RefreshToken = _tokenEngine.Create(userId, TokenType.Refresh, out _),
```

Comment:
Refresh token maybe is necessary too?

Reply:
Sounds like we need it too, but, if we update both token access token lifetime will be smaller than refresh and refresh lifetime is usless